### DISCUSSION TITLE
Problema rinnovo certificati

### DISCUSSION THREAD

**Discussion by nikpait** [indomus/forum#476] [HUB: Home Assistant]
Ciao a tutti,
non so cosa sia successo ma questo mese il certificato non si è rinnovato e mancano ormai 9 giorni.
Ho già verificato permessi e owner ma non va. 
Da log vedo questo errore:

# INFO: Using main config file /home/homeassistant/dehydrated/config
Unknown hook this_hookscript_is_broken__dehydrated_is_working_fine__please_ignore_unknown_hooks_in_your_script
Processing xxxx.duckdns.org
Unknown hook this_hookscript_is_broken__dehydrated_is_working_fine__please_ignore_unknown_hooks_in_your_script
 + Checking domain name(s) of existing cert... unchanged.
 + Checking expire date of existing cert...
 + Valid till Jul  5 12:05:17 2021 GMT Certificate will expire
(Less than 30 days). Renewing!
 + Signing domains...
 + Generating private key...
 + Generating signing request...
 + Requesting new certificate order from CA...
 + Received 1 authorizations URLs from the CA
 + Handling authorization for xxxx.duckdns.org
 + 1 pending challenge(s)
 + Deploying challenge tokens...
KO
 + Responding to challenge for xxxx.duckdns.org authorization...
Unknown hook invalid_challenge
 + Cleaning challenge tokens...
KO
 + Challenge validation has failed :(

Qualcuno può aiutarmi?

Grazie

**Comment by nikpait** [comment]
Se invece digito
./dehydrated -c

ricevo il messaggio sotto:

# INFO: Using main config file /home/homeassistant/dehydrated/config
Unknown hook this_hookscript_is_broken__dehydrated_is_working_fine__please_ignore_unknown_hooks_in_your_script
Processing xxxx.duckdns.org
Unknown hook this_hookscript_is_broken__dehydrated_is_working_fine__please_ignore_unknown_hooks_in_your_script
 + Checking domain name(s) of existing cert...unable to load certificate
3069935632:error:0909006C:PEM routines:get_name:no start line:../crypto/pem/pem_lib.c:745:Expecting: TRUSTED CERTIFICATE

**Comment by nikpait** [comment]
Risolto.

Ho eliminato i file 
cert.pem e
cert.csr

o rilanciato 
./dehydrated -c

ho ridato i permessi ai due nuovi file e certificati 
riavviato HA

Ora tutto funziona

**Reply by smarcotutti** [reply]
siccome ho un problema anche io coi certificati, dove hai trovato quei file per cancellarli? io non li trovo.
